tests/components/roomba/test_config_flow.py

```diff
@@ -206,7 +206,7 @@ async def test_form_user_discovery_and_password_fetch(hass: HomeAssistant) -> No
     assert result3["data"] == {
         CONF_BLID: "BLID",
         CONF_CONTINUOUS: True,
-        CONF_DELAY: 1,
+        CONF_DELAY: 30,
```

Comment:
I meant to import `DEFAULT_DELAY` and use it in the tests, so we do not need to update the tests if the time out is changed again.

```python
from homeassistant.components.roomba.const import DEFAULT_DELAY
```

```suggestion
        CONF_DELAY: DEFAULT_DELAY,
```